example/flutter_example/main.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I'm a little confused why this file exists.

Reply:
Me too, I'll delete it.